### PR TITLE
Add DueFocus 1.1.9

### PR DESCRIPTION
**Pull request by ran-dall**
After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not [already refused].
- [X] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

File: Casks/duefocus.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  version '1.1.9'
+  sha256 'd0e5238abf2053b4f372bf0242dc34830865d297ca81abc371cdf2bc123864d5'
+
+  url "https://web.duefocus.com/distribution/darwin/v2/DueFocus-#{version}-mac.zip"
+  name 'DueFocus'
+  homepage 'https://duefocus.com/'
+
+  app 'DueFocus.app'
+
+  uninstall delete: [
```
Comment (reitermarkus) [inline Casks/duefocus.rb]: This should be `zap trash:`.

**Comment by vitorgalvao** [comment]
We’re going to need to find an appcast.

**Comment by ran-dall** [comment]
> We’re going to need to find an appcast.

I let them know to publish the version number but the response was lukewarm, let's just close it for now.

**Comment by KorobkaVadim** [comment]
Hi Randall,

Sorry if it looked like lukewarm response. We write to Nicholas but “ He Not seen yet”
We are not hurry to  Pull requests for a specific reason:
Current v1 desktop applications Not Actual for any OS & any stores because we already rewrite a new app, with a new engine and simplified work scenarios and v2 will be released in November.
And we will use this  Pull requests  with joy and great gratitude.

Faifully yours,
DueFocus team

**Pull request closed without merging**